### ISSUE TITLE
Improve accessibility in file view and menu bar

### ISSUE THREAD

**Issue by finding-wave** [Zettlr/Zettlr#5411]
### Description

File Tree Navigation: When navigating to the file tree using the Ctrl+Shift+T shortcut, the screen reader either provides no announcement or simply says "blank" when moving between items with the arrow keys. This makes it impossible for blind users to understand the file structure and select files.

Menu Bar Access: The menu bar is not currently accessible using the Alt key, which is the standard method for screen reader users to interact with menus. This prevents blind users from accessing important functionalities available through the menu bar.

These issues severely limit the usability of the editor for people who are completely blind, as they cannot reliably navigate the file tree or access the menu bar.

### Proposed Changes

- File Tree: Ensure that the screen reader provides clear and accurate announcements of file and folder names when navigating the file tree with the arrow keys.
- Menu Bar: Implement standard Alt key navigation for the menu bar, allowing screen reader users to access and activate menu items.

### Caveats

It may cost lots of time and resources.

### Do you Wish to Attempt Implementing this Yourself?

Yes

### Zettlr Version

Stable (most recent version)

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Operating System Version

Window 11 Pro

**Comment by nathanlesage** [comment]
Thank you very much for outlining this. I always hope to improve a11y for Zettlr, so having users note where we forgot to do so is great!

Thank you also for saying you may want to implement this yourself -- if you need any help or would like to back out of implementing it yourself, just comment here!

**Comment by nathanlesage** [comment]
Absolutely, please feel free to send me an email. I would also be interested what you mean by involvement sponsorship. Let's discuss this!

**Comment by finding-wave** [comment]
@nathanlesage  I am a freshman in college taking a First-Year Experience class that requires us to volunteer in IT-related work. While searching for a markdown editor that supports live preview, I found Zettlr. Although the editor is very accessible, I found that my screen reader was unable to access the menu and file tree. I thought I could use this opportunity to contribute to this project because there aren't many accessible note-taking applications, including Obsidian, which is proprietary. I figure that if I can help make this app more accessible, then people who are screen reader users can take notes similar to their sighted counterparts. Once I have completed my hours, I would like to know if you could sign this paper stating that I have completed my volunteering hours. 

Also, I cannot find your email address, what is the best way to communicate off list?

**Comment by Prateek-coder19** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.
